### PR TITLE
feat(capture-sdk):

### PR DESCRIPTION
**Pull request by AdnanYupi**
- Camera reference documentation

PIA-2878

[PIA-2878]: https://ginis.atlassian.net/browse/PIA-2878?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java [hunk 1]
```diff
@@ -464,11 +471,6 @@ public void onInterrupted() {
         });
     }
 
-    @VisibleForTesting
```
Comment (zladzeyka) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java:467]: [Clarification] why did we remove it? Don't we use it for tests?

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentListener.java [hunk 2]
```diff
@@ -31,6 +31,16 @@ interface CameraFragmentListener {
...
     void onDocumentAvailable(@NonNull Document document);
 
+
+    /**
+     * <p>
+     * Called when starting MultiPageReview Screen
+     * </p>
+     *
+     * @param multiPageDocument the image taken by the camera or the validated imported document
+     * @param shouldScrollToLastPage if pages should scroll to the last one
```
Comment (zladzeyka) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentListener.java:41]: [Suggestion] Maybe it is worth mentioning that `shouldScrollToLastPage` is a flag that defines if pages should scroll to the last one?

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Looks good.
I'm a bit surprised to see a small refactoring in the reference documentation update 🙈.
I have only a small comment.

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java (hunk 1 above)
Comment (AdnanYupi) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java:467]: @zladzeyka I don't think we used this line of code anymore even in test cases:
` @VisibleForTesting
    PaymentQRCodeReader getPaymentQRCodeReader() {
        return mPaymentQRCodeReader;
    }`

**Review by a-szotyori — APPROVED** [review]
Thanks for also switching to lambdas! 👍

**Pull request merged**